### PR TITLE
Update to notices

### PR DESCRIPTION
**Pull request by digeff**
Update to notices

### CODE REVIEW

File: ThirdPartyNotices.txt [hunk 1]
```diff
@@ -16,6 +16,28 @@ THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
...
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Includes a Code of Conduct file adapted from the Contributor Covenant v1.4 under the license below.
+
 The MIT License (MIT)
```
Comment (roblourens) [inline ThirdPartyNotices.txt]: Is there supposed to be a project name above this or is this also related to InversifyJS? What happened to Ninject?

Comment (digeff) [reply ThirdPartyNotices.txt]: We'll leave it under InversifyJS directly
